modules/preview-react/pill/stories/Pill.stories.mdx

```diff
@@ -0,0 +1,237 @@
+import {Meta} from '@storybook/addon-docs';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {Pill} from '@workday/canvas-kit-preview-react/pill';
+
+import {Basic} from './examples/Basic';
+import {ClickablePillWithAvatar} from './examples/WithAvatar';
+import {ClickablePillWithIcon} from './examples/WithIcon';
+import {ClickablePillWithCount} from './examples/WithCount';
+import {RemovablePill} from './examples/WithRemovable';
+import {ListOfPills} from './examples/WithList';
+
+<Meta title="Preview/Pill/React" component={Pill} />
+
+# Canvas Kit Pill
+
+Pills are static or interactive elements that allow users to input, filter, or label information.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+Pills are used to visually label objects on a page for quick recognition. They’re offered as both
+static (read-only) and interactive elements. They allow users to filter a list or table, or label
+information to help with scanning and organization.
+
+### Basic Example
+
+If the Pill has `variant='readOnly'`, it will look like a read-only Pill. This is a non-interactive
+element that is used to display information.
+
+> **_NOTE:_** By default, the Pill's width will max out at `200px` and the text will be truncated
+> with an ellipsis and render an OverflowTooltip on hover and focus. This max width can be changed
+> by providing a `maxWidth` prop on the Pill.
+
+<ExampleCodeBlock code={Basic} />
+
+### Interactive Pills
+
+All leading elements (icons or avatars) are intended to be descriptive, helping support the label.
+Do not rely on the leading element to indicate the interaction behavior. By default a Pill is
+considered interactive, therefore it's default variant is `default`.
+
+```tsx
+<Pill onClick={() => console.warn('clicked')}>
+  <Pill.Avatar />
+  Regina Skeltor
+</Pill>
+```
+
+#### Avatar
+
+The Avatar should appear before the text of a Pill. You have access to all Avatar props.
+
+<ExampleCodeBlock code={ClickablePillWithAvatar} />
+
+#### Icon
+
+The `Pill.Icon` icon will default to rendering a `plusIcon`. You can change this by providing a
+`icon` prop to `Pill.Icon`
+
+<ExampleCodeBlock code={ClickablePillWithIcon} />
+
+#### Count
+
+The count appears after the label. It is usually associate to the label. If you have a category, the
+count will dirrectly correlate to that category.
+
+<ExampleCodeBlock code={ClickablePillWithCount} />
+
+### Removable Pills
+
+Removable Pills display an `X` icon after the label. They have a smaller, more specific focus state
+and click target to be more intentional about their actions and to avoid unintended removal of a
+Pill.
```

Comment:
```suggestion
Removable `Pill`s display an `X` icon after the label. They have a smaller, more specific focus state
and click target to be more intentional about their actions and to avoid unintended removal.
```